docs/02.API-REFERENCE.md

```diff
@@ -166,9 +166,14 @@ RegExp object optional flags:
 
   - JERRY_REGEXP_FLAG_GLOBAL - global match; find all matches rather than stopping after the first match
   - JERRY_REGEXP_FLAG_IGNORE_CASE - ignore case
-  - JERRY_REGEXP_FLAG_MULTILINE - multiline; treat beginning and end characters (^ and $) as working over
-  multiple lines (i.e., match the beginning or end of each line (delimited by \n or \r), not only the
-  very beginning or end of the whole input string)
+  - JERRY_REGEXP_FLAG_MULTILINE - multiline; treat beginning and end characters (^ and $) as working
+    over
+    multiple lines (i.e., match the beginning or end of each line (delimited by \n or \r), not only the
+    very beginning or end of the whole input string)
+  - JERRY_REGEXP_FLAG_STICKY - The sticky flag indicates that it matches only from the index indicated by the lastIndex property
+  - JERRY_REGEXP_FLAG_UNICODE - The unicode flag enables various Unicode-related features
+  - JERRY_REGEXP_FLAG_DOTALL -The dotall flag indicates that the dot special character (".") should additionally match the following line terminator ("newline") characters in a string;
+
 
 *New in version 2.0*.
```

Comment:
Please add a line here:
*Changed in version [[NEXT_RELEASE]]* : Added `JERRY_REGEXP_FLAG_MULTILINE`, .... values.